presto-docs/src/main/sphinx/develop/functions.rst

```diff
@@ -112,11 +292,8 @@ average of a ``DOUBLE`` column.
   aggregation. It takes the final state object (the result of merging all
   partial states) and writes the result to a ``BlockBuilder``.
 
-* Where does serialization happen, and what is ``@GroupedAccumulatorState``?
+* Where does serialization happen?
 
   The ``@InputFunction`` is usually run on a different worker from the
   ``@CombineFunction``, so the state objects are serialized and transported
-  between these workers by the aggregation framework. ``@GroupedAccumulatorState``
```

Comment:
That `@` is a typo. It was always an interface like `AccumulatorState`. I think it's worth keeping, but yes lets fix the typo :)